share/locale/musescore_zh_HK.ts

```diff
@@ -14243,7 +14243,7 @@ Fret %1 on strings %2</source>
         <location filename="../../src/engraving/dom/spanner.cpp" line="1594"/>
         <location filename="../../src/notation/internal/notationaccessibility.cpp" line="181"/>
         <source>End beat: %1</source>
-        <translation>末尾拍</translation>
+        <translation>末尾拍: %1</translation>
```

Comment:
For "Start beat: %1", they use a different kind of colon, isn't that necessary here too?

Reply:
OK, fixed.